extmod/modubinascii.c

```diff
@@ -89,14 +88,89 @@ mp_obj_t mod_binascii_unhexlify(mp_obj_t data) {
 }
 MP_DEFINE_CONST_FUN_OBJ_1(mod_binascii_unhexlify_obj, mod_binascii_unhexlify);
 
+static const char *base64codes = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/=";
```

Comment:
It certainly is fun to work in an environment where the usual assumptions aren't true, like "it's always a good idea to sacrifice compactness for speed". Rewritten implementation 52 bytes smaller. I'll see if I can squeeze a bit more out by looking at that switch with the redundant code.